crates/stackable-operator/src/commons/s3/crd.rs

```diff
@@ -98,18 +98,31 @@ pub enum S3AccessStyle {
 #[derive(Clone, Debug, Deserialize, Eq, JsonSchema, PartialEq, Serialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Region {
-    #[serde(default = "default_region_name")]
+    #[serde(default = "Region::default_region_name")]
     pub name: String,
 }
 
+/// Having it as `const &str` as well, so we don't always allocate a [`String`] just for comparisons
+const DEFAULT_REGION_NAME: &str = "us-east-1";
+impl Region {
```

Comment:
The const can be moved into the impl, and also made pub in case it is needed (eg: in a warning).

```suggestion
impl Region {
    /// Having it as `const &str` as well, so we don't always allocate a [`String`] just for comparisons
    pub const DEFAULT_REGION_NAME: &str = "us-east-1";

```

Then use it as `Self::DEFAULT_REGION_NAME`

Reply:
Makes sense, thanks!